R/aldex.R

```diff
@@ -93,74 +86,30 @@ aldex <- function(Y, X, data=NULL, nsample=2000,  scale=NULL,
     formula <- NULL
   }
 
-  ## perform streaming 
   out <- list()
-  nsample.remaining <- nsample
   iter <- 1
-  if (stream) {
-    nsample.local <- floor(streamsize*100000/(N*D*8))
-    if (nsample.local < 1) stop("streamsize too small")
-  } else {
-    nsample.local <- nsample
+  chunk.sizes <- aldex.getchunksizes(Y, X, nsample, streamsize)
+  for(chunk.size in chunk.sizes) {
```

Comment:
please fix error by just updating logic in my code. It should be fixed by just adding a simple if else statement (if nsample.remaining < nsample.local, then use nsample.remaining instead, etc...). 

You can minimize the changes needed to fix the error. You also don't need to change so many variable names along the way. I cannot see a reason to shift to this new "chunk" naming scheme. It just makes it hard for a maintainer to review PRs.